src/perf_event/mod.rs

```diff
@@ -7,4 +7,348 @@ pub mod tracing;
 use crate::syscall::bindings::perf_event_attr;
 pub use event::*;
 
-type RawAttr = perf_event_attr;
+#[derive(Debug, Clone)]
+pub struct RawPerfEventAttr(perf_event_attr);
```

Comment:
I did consider `PerfEventAttr` as the struct name, but soon I realized that the name of a variable that's instantiated from `PerfEventAttr` may be literally same with repr(C) struct `perf_event_attr`. 

We need to make a trade off here, I think changing all `PerfEventAttr` variables' names to `event_attr` may be is acceptable.

Reply:
"Since users may use Config::from_raw to unsafely construct their Config, it is necessary to provide public access to the RawPerfEventAttr field to construct RawPerfEventAttr."

I don't get it. We'd better to have a face to face discussion.